<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WriteStationTBL</class>
 <widget class="QDialog" name="WriteStationTBL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>550</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Current Configurations:</string>
   </property>
   <widget class="QLabel" name="label_currentEth0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>341</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>current Eth0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentEth1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>321</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>current Eth1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentTodID">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>current TodID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentSubnet">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>current Subnet:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentTodPort">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>current TodPort:</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>781</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="TabConfig">
    <attribute name="title">
     <string>Setting</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>771</width>
       <height>321</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_CarID">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Car ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Terminal">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Terminal A/B</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_CarID">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Terminal">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_IP_eth0">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>321</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Eth0</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_IP_eth1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>321</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Eth1</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_TodID">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>321</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Tod ID</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_Subnet">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>321</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Subnetwork</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>250</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>SAVE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_CarIDUp">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>0</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>UP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_CarIDDown">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_CarTermDown">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>80</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_CarTermUp">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>0</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>UP</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="TabUpdate">
    <attribute name="title">
     <string>Update</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_Update">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Update_Intro">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>621</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>Files from USB drive will be paste into display and replace the old files. </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>190</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>clean</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WriteStationTBL</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WriteStationTBL</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
